package.json

```diff
@@ -22,9 +22,9 @@
     "@typescript/vfs": "1.6.0",
     "@web/dev-server-esbuild": "1.0.2",
     "@web/test-runner": "0.19.0",
-    "@zenfs/core": "0.9.7",
-    "@zenfs/dom": "0.2.6",
-    "@zenfs/zip": "0.3.1",
+    "@zenfs/core": "^1.0.0",
+    "@zenfs/dom": "^1.0.0",
+    "@zenfs/zip": "^0.5.0",
```

Comment:
looks like this upgrade impacts the size. can you bump up the size of the dist/play-pen.js's none field below to 26679 KB? you can test locally by running `npm run test:size`.

Reply:
Just FYI, you can save some bundle size by not installing the optional dependencies from `@zenfs/dom`. They are only used for unit testing.

Reply:
Done in [`25dd63b` (#40)](https://github.com/reddit/play/pull/40/commits/25dd63b864855c0082fad34caf87a93c3a4e143e).

Reply:
thanks! if you see a needless dependency, zapping it would be welcome. looks like the UI tests are failing now:

```
$ npm run test:ui

src/elements/play-assets-dialog.test.ts:

 🚧 Browser logs:
      SyntaxError: The requested module './index.js' does not provide an export named 'default'

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 


src/elements/play-pen-header.test.ts:

 🚧 Browser logs:
      SyntaxError: The requested module './index.js' does not provide an export named 'default'

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 


src/elements/play-assets/play-assets-local-archive.test.ts:

 🚧 Browser logs:
      SyntaxError: The requested module './index.js' does not provide an export named 'default'

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 


src/elements/play-assets/play-assets-local-directory.test.ts:

 🚧 Browser logs:
      SyntaxError: The requested module './index.js' does not provide an export named 'default'

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 


src/elements/play-assets/play-assets-virtual-fs.test.ts:

 🚧 Browser logs:
      SyntaxError: The requested module './index.js' does not provide an export named 'default'

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 


src/elements/play-assets/play-assets.test.ts:

 🚧 Browser logs:
      SyntaxError: The requested module './index.js' does not provide an export named 'default'

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 
```